docs/new/references/flashblocks.md

```diff
@@ -0,0 +1,148 @@
+---
+description: Flashblocks support (alpha feature)
+---
+
+New support for "Flashblocks" is now available for alpha testing on Base Mainnet.
+
+ref: https://docs.base.org/base-chain/flashblocks/apps
+
+## Disclaimers
+
+* The only endpoint supporting Flashblocks is `https://base-mainnet-flash.streamingfast.io:443`.
+* That endpoint is not guaranteed to be stable or available at all times.
+* The protocol might still change in the next few weeks, as we gather feedback on usage.
+* Flashblocks coming from Firehose or Substreams endpoint should not be considered final data. Only write to database data coming from full blocks: there is no "undo" mechanism for Flashblocks.
+
+## Description
+
+* Flashblocks are partial blocks that are not fully confirmed yet. 
+* They are emitted every 200ms and contain a fraction of the transactions that will be in the final block.
+
+### Flashblocks in Substreams
+
+#### Partial Blocks
+
+* In Substreams, Flashblocks are called **partial blocks**, as a generalization of the concept, even though Flashblocks are the only supported implementation yet.
+* To benefit from partial blocks: 
+  1. you need the latest version of substreams CLI or library.
+  2. your substreams modules should avoid doing "block-level aggregations" and should only work on what is inside the "transactionTraces"
```

Comment:
Don't see the point to bullet points here (the parent ones, the child ones `1., 2.` are fine.)

Reply:
Also it's usually more approriate to have all numbered list as 1., the Markdown rendered takes care of putting the right numbers, make it easier to edit and refactor.